nvflare/fuel/f3/drivers/uds_driver.py

```diff
@@ -86,6 +84,7 @@ def connect(self, connector: ConnectorInfo):
         self.add_connection(connection)
         connection.read_loop()
         self.close_connection(connection)
+        self.remove_socket_file(socket_path)
```

Comment:
Will both the "connector" side and "listener" side try to remove the same socket file at the same time?

Reply:
It may happen. Only last one closes the connection can remove it. This could be either side so both sides try to close. Only one side will succeed.